PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,34 +1,37 @@
-## Description of Work
-<!--add description of work here 🙏🏼 -->
+## Description
+<!--add a short description of work here 🙏🏼 -->
 
 ### Change List
-
-- [] item 1
-- [] item 2
-
-## Deployment Notes
-
-### Deployment Checklist
-
-`select all that apply`
-
-### Code changes
-
-- [] Unit/Integration tests added
-- [] README updated
-- [] `/docs` updated
-
-### Infra Changes
-
-- [] All environments updated
-- [] Applicable Lucidcharts updated
-- [] If modules were moved were resources cleaned up
-
-## Technical Debt
+<!-- More detailed list of the major changes that will help explain what is going on with your PR -->
+- item 1
+- item 2
+
+<details>
+<summary>Extra Stuff</summary>
+  <!-- select all that apply -->
+  <!-- More details from GitHub -->
+  <!-- https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax -->
+  <!-- https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting -->
+        
+  ### Diagrams
+  <!-- If a diagram would be more helpful to explain a new flow or change, consider adding that here ->>
+  <!-- https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams -->   
+  
+  ### Proof of Work
+  <!-- Add a screenshot, curl command, or other items that show reviewers what the outcome of your code changes are --> 
+  
+  ### Code Changes
```

Comment:
I found myself just adding code changes and infra changes to the change list and deleting these from the PR template. I think the code diff supplements any details missing from the list of changes at the top.

Reply:
yeah that's a good point, I don't use these sections very frequently either

Reply:
I get that... I made this section collapsable too for that reason